src/bindings/dom/html.js

```diff
@@ -0,0 +1,65 @@
+import { Localization } from './base';
+import { overlayElement } from './overlay';
+
+export { contexts } from './base';
+
+const allowed = {
+  elements: [
+    'a', 'em', 'strong', 'small', 's', 'cite', 'q', 'dfn', 'abbr', 'data',
+    'time', 'code', 'var', 'samp', 'kbd', 'sub', 'sup', 'i', 'b', 'u',
+    'mark', 'ruby', 'rt', 'rp', 'bdi', 'bdo', 'span', 'br', 'wbr'
+  ],
+  attributes: {
+    global: ['title', 'aria-label', 'aria-valuetext', 'aria-moz-hint'],
+    a: ['download'],
+    area: ['download', 'alt'],
+    // value is special-cased in isAttrAllowed
+    input: ['alt', 'placeholder'],
+    menuitem: ['label'],
+    menu: ['label'],
+    optgroup: ['label'],
+    option: ['label'],
+    track: ['label'],
+    img: ['alt'],
+    textarea: ['placeholder'],
+    th: ['abbr']
+  }
+};
+
+export class HTMLLocalization extends Localization {
+  overlayElement(element, translation) {
+    return overlayElement(this, element, translation);
+  }
+
+  // XXX the allowed list should be amendable; https://bugzil.la/922573
+  isElementAllowed(element) {
+    return allowed.elements.indexOf(element.tagName.toLowerCase()) !== -1;
```

Comment:
we should only lowercase in HTML documents for HTML Elements.

Also, tagName or localName? Do we have prefixed markup in l10n?